AzureADSample/AzureADSample/View/SettingsPage.xaml.cs

```diff
@@ -0,0 +1,17 @@
+using AzureADSample.ViewModel;
+
+namespace AzureADSample.Views;
+
+public partial class SettingsPage : ContentPage
+{
+	public SettingsPage(SettingsPageViewModel vm)
+	{
+		InitializeComponent();
+		BindingContext = vm;
+	}
+
+	async void LogOutBtn(object sender, EventArgs e)
+    {
+		// Gal -> add code here .. 
```

Comment:
Could you create a new issue for this to make sure we're tracking this TODO?

Reply:
It's been created: https://github.com/rachelkang/AuthenticationSamples/issues/2